tests/drivers/clock_control/clock_control_api/testcase.yaml

```diff
@@ -13,6 +13,7 @@ tests:
       - sltb010a@0
       - xg24_dk2601b
       - xg27_dk2602a
+      - xg29_rb4412a
```

Comment:
This board is introduced in the next commit.

Reply:
Fixed, good catch!